src/component/mxgraph/config/ShapeConfigurator.ts

```diff
@@ -51,6 +53,9 @@ export default class ShapeConfigurator {
     mxCellRenderer.registerShape(ShapeBpmnElementKind.TASK_SEND, SendTaskShape);
     // artifacts
     mxCellRenderer.registerShape(ShapeBpmnElementKind.TEXT_ANNOTATION, TextAnnotationShape);
+
+    // shapes for flows
+    mxCellRenderer.registerShape(ShapeBpmnEventKind.MESSAGE, MessageShape);
```

Comment:
:warning: I clearly dislike using something related to message event to identify something related to message flow. This mixes concepts and is confusing
Message flow icon? the value `message` is not really meaningful

Reply:
Finally, I replaced `message` by `messageFlowIcon`